sqlmypy.py

```diff
@@ -369,10 +466,18 @@ class User(Base):
             # Something complex, stay silent for now.
             new_arg = AnyType(TypeOfAny.special_form)
 
+    # use private api
+    current_model = ctx.api.scope.active_class()  # type: ignore # type: TypeInfo
```

Comment:
Put `# type: ignore` at the end, after type comment, otherwise the type comment will be ignored.